fivetran_provider_async/example_dags/example_fivetran_async.py

```diff
@@ -22,19 +24,21 @@
 with dag:
     fivetran_async_op = FivetranOperatorAsync(
         task_id="fivetran_async_op",
-        connector_id="bronzing_largely",
+        connector_id="{{ var.value.connector_id }}",
     )
 
+    delay_task = PythonOperator(task_id="delay_python_task", python_callable=lambda: time.sleep(180))
```

Comment:
why do we need this? it was running fine for me without adding any delay

Reply:
@phanikumv on second thought, I think it would be better to have 2 separate example DAGS. 1 showing how to use the FivetranOperator -> FivetranSensorAsync and 1 for just the FivetranOperatorAsync.